### DISCUSSION TITLE
Dark Mode Support

### DISCUSSION THREAD

**Discussion by gfeeng** [greatfrontend/greatfrontend#4] [Ideas/Feature Requests]
Everyone loves Dark Mode, including us. Would you like to have Dark Mode support too? Leave a comment and upvote this discussion if so!

Add some examples of sites that has dark mode you enjoy using!

**Comment by pratapbunga** [comment]
I've been using the platform for quite a bit now. Love to see Dark Mode enabled.

**Comment by crypto-dot** [comment]
Dark mode gang

**Comment by 2D9U** [comment]
It would help a lot in terms of user experience

**Comment by hectoraldairah** [comment]
I think the dark mode could be a nice add.
The color scheme of the actual design system seems to be very efficient in dark mode.

**Comment by mohitvirli** [comment]
Any updates on the same? 

Edit - Night Eye chrome extension works alright

**Comment by yangshun** [comment]
We just launched dark mode, hope you all like it!

<img width="1552" alt="image" src="https://github.com/greatfrontend/greatfrontend/assets/1315101/5df1a763-9d0c-404a-aae3-e3eac643591f">
